### PR TITLE
Added the basic outline of the project

### PR DESCRIPTION
**Pull request by rohithasrk**
Added all the shared code as mentioned in #1. All I have to do is write tests now. I can start writing them but I've not figured out writing modular tests yet, i.e., for ex., A test case for `MultitenantAdminMixin` which can be extended.

### CODE REVIEW

File: openwisp_utils/tests/utils.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+from django.contrib.auth.models import Permission
+from django.db.models import Q
+from django.urls import reverse
+
+from openwisp_users.models import OrganizationUser
+
+user_model = get_user_model()
+
+
+class TestAdminMixin(object):
```
Comment (nemesisdesign) [inline openwisp_utils/tests/utils.py]: let's rename it to `TestMultitenantAdminMixin`

Comment (rohithasrk) [inline openwisp_utils/tests/utils.py]: Yeah. Okay

File: tests/test_project/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+__dependencies__ = []
```
Comment (nemesisdesign) [inline tests/test_project/__init__.py]: remove this line

File: tests/test_project/tests/test_staticfinders.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import unittest
+
+from openwisp_utils.staticfiles import DependencyFinder
+
+
+class TestStaticFinders(unittest.TestCase):
+    def test_dependency_finder(self):
+        finder = DependencyFinder()
+        self.assertIsInstance(finder.locations, list)
```
Comment (nemesisdesign) [inline tests/test_project/tests/test_staticfinders.py]: change to `self.assertIn('django_netjsonconfig', finder.locations[0][1]` and do something similar for `DependencyLoader`

Comment (rohithasrk) [inline tests/test_project/tests/test_staticfinders.py]: Okay.

**Comment by rohithasrk** [comment]
This PR references #3 too. There are some test fail issues that we're facing. Can we create a new issue @nemesisdesign?

**Comment by nemesisdesign** [comment]
@rohithasrk yes

**Pull request merged**